src/test/java/org/sonatype/repository/conan/internal/proxy/ConanProxyRecipeTest.java

```diff
@@ -61,7 +57,7 @@ public void tearDown() throws Exception {
   @Test
   public void canMatchOnDownloadUrls() {
     when(request.getPath()).thenReturn("/v1/conans/jsonformoderncpp/2.1.1/vthiery/stable/download_urls");
-    assertTrue(downloadUrlsMatcher().matches(context));
+    //assertTrue(downloadUrlsMatcher().matches(context));
```

Comment:
If these are _dead_ dead (and not just _mostly_ dead) then I'd remove them rather than just comment them out.

Reply:
https://github.com/sonatype-nexus-community/nexus-repository-conan/pull/15/commits/37e24362caf1ef4c1241e8fe783afc948d3e8206